web/src/components/pages/dashboard/dashboard.tsx

```diff
@@ -97,7 +112,7 @@ const TopBar = ({ dashboardLocale }: { dashboardLocale: string }) => {
         </LocaleNavLink>
       </nav>
 
-      <div className="languages">
+      <div className={`languages ${isChallengeTabVisible ? 'challenge' : ''}`}>
```

Comment:
The challenge pilot is going to be English-only. I think it makes sense to prevent showing the language bar for the Challenge tab, and instead put in some static text that says "Language: ENGLISH" like in the mock-ups.